src/pages/RequisitionsPage.js

```diff
@@ -31,7 +31,8 @@ export class RequisitionsPage extends React.Component {
       isRequestView: true,
       isCreatingRequisition: false,
     };
-    this.requisitions = props.database.objects('Requisition');
+    this.requisitions = props.database.objects('Requisition')
+                                      .filtered('serialNumber != "-1"');
```

Comment:
@edmofro If you're wondering why we're back to doing this in the Requisition page:

SyncPostProcessor uses the UIDatabase, so when trying to find records with serialNumber '-1' it'd never find any because they were already filtered out.

I suppose a different solution would be make another switch option in UIDatabase for 'RawRequisitions' or something that doesn't do any filtering.

Or another function objectsDirect(type) that just passes straight to the realm objects without any filters.